clang/include/clang/Driver/Options.td

```diff
@@ -4047,6 +4047,10 @@ def fsycl_libspirv_path_EQ : Joined<["-"], "fsycl-libspirv-path=">,
   Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>, HelpText<"Path to libspirv library">;
 def fno_sycl_libspirv : Flag<["-"], "fno-sycl-libspirv">, Visibility<[ClangOption, CLOption, DXCOption]>,
   HelpText<"Disable check for libspirv">;
+def fsycl_use_spirv_backend_for_spirv_gen : Flag<["-"], "fsycl-use-spirv-backend-for-spirv-gen">,
+  Visibility<[ClangOption, CLOption]>, HelpText<"Use SPIR-V backend for SPIR-V code generation. "
```

Comment:
Should this be `HelpHidden`?

Reply:
I think I will wait to hear from @mdtoguchi to see if we need to document this. If not, then we can add the HelpHidden here. Thanks